omise/views/templates/hook/payment.tpl

```diff
@@ -68,3 +68,66 @@
     </p>
   </div>
 </div>
+
+<script src="https://cdn.omise.co/omise.js.gz"></script>
+
+<script>
+  const omiseCheckout = function omiseCheckout() {
+    if (typeof Omise === 'undefined') {
+      alert('{l s='Unable to process the payment, loading the external card processing library is failed. Please contact the merchant.' mod='omise'}');
+      return false;
+    }
+
+    omiseLockCheckoutForm(omiseCheckoutForm);
+
+    const card = {
```

Comment:
@nimid after we have https://github.com/omise/omise-prestashop/pull/12/files#diff-703d934df9e134d3b80f37155717d048R104

This object seems ok to use it also.